src/style.css

```
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  margin: 0;
+  padding: 0;
+}
+
+.form {
+  height: 70px;
+  max-width: 100%;
+  display: block;
+  position: relative;
+  border-radius: 4px;
+  border: none;
+  margin-right: 8px;
+  margin-left: 8px;
+  margin-top: 20px;
```

Use margin shorthand
```suggestion
  margin-right: 8px;
  margin-left: 8px;
  margin-top: 20px;
  margin: 20px 8px 0;
```